action.yml

```
@@ -20,6 +20,8 @@ inputs:
 outputs:
   tf_actions_plan_has_changes:
     description: 'Whether or not the Terraform plan contained changes.'
+  tf_outputs:
+    description: 'The outputs in json format from Terraform state.'
```

Being a bit more concise.
```suggestion
    description: 'The Terraform outputs in JSON format.'
```